src/mlpack/methods/ann/layer/reparametrization.hpp

```diff
@@ -71,6 +71,18 @@ class Reparametrization
                     const bool stochastic = true,
                     const bool includeKl = true,
                     const double beta = 1);
+    
+  //! Copy Constructor 
+  Reparametrization(const Reparametrization& layer);
+    
+  //! Move Constructor 
+  Reparametrization(Reparametrization&& layer);
+    
+  //! Copy assignment operator 
```

Comment:
```suggestion
  //! Copy Constructor.
  Reparametrization(const Reparametrization& layer);
    
  //! Move Constructor.
  Reparametrization(Reparametrization&& layer);
    
  //! Copy assignment operator.
```

Add missing stop at the end.